JuntosFinanzas/scivias
# integrate-amazon-cognito-with-scivias

This addresses [JSP-335](https://juntos.atlassian.net/browse/JSP-335)

#### Screenshots
<img width="1245" alt="Screen Shot 2020-04-10 at 12 31 32 AM" src="https://user-images.githubusercontent.com/28448244/78949157-b230ad00-7ac2-11ea-873e-7f88ed1fa82c.png">

#### How to test
- on this branch, remove:  `if Rails.env.production?` in the `SingleSignOnAuthentication` module.
- visit the index route: `http://localhost:3000`
- you should see a link to `Sign in` which should redirect you to the test Juntos Cognito login URL. After signing in, it should prompt you to change your password. Then, it should redirect you to the Scivias index page. 
- On the home page, you should see your email and a link to sign out.
